packages/aggregator/src/index.ts

```diff
@@ -11,7 +11,7 @@ export function createAggregatorLoader(handlers: HandlerList): Loader {
       page.blocks.map(async block => {
         const handler = handlerRegistry(block.type);
 
-        const result = await handler(block, ctx);
+        const result = await handler(block, ctx, page);
```

Comment:
I agree it’s a danger. In this case, I was deliberately copying the loader signature and wanted to avoid introducing a non-obvious difference. But open for reconsidering this. Did you have something in mind?